main/src/components/SignupForm.js

```diff
@@ -50,7 +50,7 @@ class SignupForm extends React.Component {
 
```

Comment:
This file seems fine. The lines that were added seem to be the exact same as what was supposedly removed.